applications/StructuralMechanicsApplication/custom_python/structural_mechanics_python_application.cpp

```diff
@@ -171,6 +172,10 @@ PYBIND11_MODULE(KratosStructuralMechanicsApplication,m)
     /* Point moment */
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, POINT_MOMENT )
 
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, AXIAL_FORCE )
```

Comment:
I think we should consider the same format considered in the existing beams

Reply:
@AlejandroCornejo this does not exist in the existing beams right?